Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -760,13 +763,21 @@ def create_text_out_format(iocs: IO, request_args: RequestArguments) -> Union[IO
             new_line = '\n'
     iocs.close()
     if len(ipv4_formatted_indicators) > 0:
+        if extensive_logging:
+            demisto.debug(f"edl: Original IPv4 indicators: {ipv4_formatted_indicators}")
```

Comment:
in a separate log line, please print the length of Original ipv4_formatted_indicators.
this should be printed regardless of extensive_logging

also, is it original if the var called formatted?

Reply:
yes, it is before the format